ibis/expr/api.py

```diff
@@ -236,7 +235,24 @@
 NA = null()
 
 
+def sequence(values):
+    """
+    Wrap a list of Python values as an Ibis sequence type
+
+    Parameters
+    ----------
+    values : list
+      Should all be None or the same type
+
+    Returns
+    -------
+    seq : Sequence
+    """
+    return ops.ValueList(values).to_expr()
+
+
 def schema(pairs=None, names=None, types=None):
+    # TODO(kszucs): if isinstance(pairs, dict): Schema.from_dict
```

Comment:
Going to create a ticket to reuse this schema function from `ibis.table()` for a more unified experience.